### ISSUE TITLE
Cannot run sgd example

### ISSUE THREAD

**Issue by ptoulis** [airoldilab/sgd#45]
Here is what i get when I run the "Dobson" example. 

Error in rep(no, length.out = length(ans)) : 
  attempt to replicate an object of type 'closure'

My version is 
platform       x86_64-pc-linux-gnu         
arch           x86_64                      
os             linux-gnu                   
system         x86_64, linux-gnu           
status                                     
major          3                           
minor          0.2                         
year           2013                        
month          09                          
day            25                          
svn rev        63987                       
language       R                           
version.string R version 3.0.2 (2013-09-25)
nickname       Frisbee Sailing

**Comment by dustinvtran** [comment]
Seem like it's specific to the R version. I was able to get the same error after `module load centos6/R-3.0.2` in Odyssey but using `module load centos6/R-3.1.1` it works.

I spotted the error, which is somehow inherent with how `ifelse()` does things differently in the two versions. I switched them to the standard if and else. See 017a6e1a6b3f9fc86731e5b59afca1468d971502

See if it works now on the updated package
```{bash}
devtools::install_github("airoldilab/sgd")
```
